README.md

```diff
@@ -42,6 +39,20 @@ await save('./features/dist/login.feature', ast, {
 });
 ```
 
+### License text
+
+You can set the license text to the feature files in 3 ways:
+
+1. **License file** - you can set the `licenseFile` option to the file's path where the license is stored, then it is added to each feature file as a comment.
+2. **License test** - yan can set the `licenseText` option to the exact license text, and it is added to each feature file as a comment.
+3. If **both** a license file and a license text is set, then the tool will merge these and add to each feature file. **Important**, that to merge these, the tool will use the `${LICENSE}` token in the license text, to inject the content of the license file, thus it must be added to the license text!
+
+> You can find a proper **License** in the following site for your project: https://choosealicense.com/licenses.
+
+### License placement
+
+The license text can be placed either in the start or the end of the feature file. By default it is placed in the start of the feature file. It can be set via the `placement` coonfiguration option, with one of the `LicensePlacement` enum values.
```

Comment:
```suggestion
The license text can be placed either in the start or the end of the feature file. By default it is placed in the start of the feature file. It can be set via the `placement` configuration option, with one of the `LicensePlacement` enum values.
```